transactions/templates/pendingTransactions.html

```diff
@@ -22,28 +22,36 @@
   </script>
 {% endblock %}
 {% block content %}
```

Comment:
This file should come in internal_users app, right?

Reply:
If transaction is a different app then this is the right place, if transaction will be clubbed with internal users then this should be moved.